dev/ci/scripts/wolfi/apko-check-lock.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -euf -o pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")/../../../.."
+REPO_DIR=$(pwd)
+
+BRANCH="${BUILDKITE_BRANCH:-'default-branch'}"
+MAIN_BRANCH="main"
+IS_MAIN=$([ "$BRANCH" = "$MAIN_BRANCH" ] && echo "true" || echo "false")
+
```

Comment:
Almost forgot about this - since agents can run multiple builds we need to execute this to ensure the agent is OK otherwise we might hit an issue where it will wait on another bazel build
```suggestion
echo "~~~ :aspect: :stethoscope: Agent Health check"
/etc/aspect/workflows/bin/agent_health_check
```

Reply:
Added, thanks